sdk/communication/Azure.Communication.Sms/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1.0.0-beta.4 (Unreleased)
 
+###Added
+- Support for creating PhoneNumberAdministrationClient with TokenCredential
```

Comment:
I think you meant sms here

Reply:
Great catch!! Thank you, I've fixed this